package.json

```diff
@@ -160,6 +160,7 @@
     "ts-node": "10.9.2",
     "tsconfig-paths-webpack-plugin": "3.5.2",
     "typescript": "5.4.3",
+    "uuid": "^9.0.1",
```

Comment:
Vi har en `generateUUID()` man evt kunne brukt i stedet for å ta inn bibliotek her.